events/dvg_atlantia.txt

```diff
@@ -0,0 +1,794 @@
+			add_modifier = {
+			name = dvg_demoratic_miltias
+			days = 1825
+			is_decaying	= yes
+		}
+		
+			
+		}
+		
+	}
+
+	option = {
+		name = dvg_atlantia.4.a
+		default_option = yes
+		
+		add_journal_entry = {
+			type = je_reconquering_atlantia
+		}
+		
+		c:LOT = {
+		 
+		 create_diplomatic_play = {
+			 name = reconquering_atlantia.1.t
+		
+			target_state = s:STATE_COTE_DU_CAFE.region_state:AMA
+			
+			war = no
+			
+			 type = dp_return_state
+			
+			 add_war_goal = {
+				 holder = c:AMA
+				 type = war_reparations
+				 target_country = c:LOT
+			 }
+			
+			 add_war_goal = {
+				 holder = c:AMA
+				 type = humiliation
+				 target_country = c:LOT
+			 }
+		 }
+	 }
+		
+		add_loyalists = {
+			value = 0.25
+			culture = cu:lotharian
+		}
+		change_variable = {
+				name = aigle_var
+				add = 5
+			}
+		add_radicals = {
+			value = 0.25
+			culture = cu:atlantian
+		}
+		change_relations = {
+				country = c:AMA
+				value = -25
+			}
+		ai_chance = {
+			base = 80
+		}
+	}
+	option = {
+		name = dvg_atlantia.4.b
+		default_option = yes
+		change_relations = {
+				country = c:AMA
+				value = 60
+			}
+		change_variable = {
+				name = aigle_var
+				add = -5
+			}
+		ig:ig_armed_forces = {
+			add_modifier = {
+			name = modifier_army_dissaproves
+			days = 1825
+		}
+			
+	}
+	ai_chance = {
+			base = 20
+		}
+	
+}
+}
+
+dvg_atlantia.5 = {
+	type = country_event
+	placement = ROOT
+
+	title = dvg_atlantia.5.t
+	desc = dvg_atlantia.5.d
+	flavor = dvg_atlantia.5.f
+	
+	event_image = {
+		video = "gfx/event_pictures/europenorthamerica_capitalists_meeting.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/middleeast/courtroom_upheaval"
+
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+
+	duration = 3
+
+	trigger = {
+	}
+	
+	
+	# Liberte!
+	option = {
+		name = dvg_atlantia.5.a
+		default_option = yes
+		
+		change_relations = {
+				country = c:ATL
+				value = 50
+			}
+		
+		c:NEA = {
+		set_owes_obligation_to = {
+			country = c:ATL
+			setting = yes
+		}
+		
+		
+		
+		add_modifier = {
+			name = dvg_atlantia_support
+			days = 1825
+			is_decaying	= yes
+		}
+		
+		activate_law = law_type:law_national_guard
+		}
+		c:ATL = {
+		create_diplomatic_pact = {
+			country = c:NEA
+			type = protectorate
+		}
+		add_modifier = {
+			name = dvg_atlantia_giving_support
+			days = 1825
+			is_decaying	= yes
+		}
+		}
+		set_global_variable = {
+				name = atlantia_independent
+				value = 0
+			}
+		set_global_variable = nouvellsatian_support
+		ai_chance = {
+			base = 40
+		}
+		c:LOT = { trigger_event = dvg_atlantia.9 
+				
+		
+		}
+		s:STATE_COTE_DU_CAFE = {
+			add_claim = c:LOT
+		}
+		ai_chance = {
+			base = 80
+		}
+	}
+	
+	option = {
+		name = dvg_atlantia.5.b
+		
+		add_loyalists = {
+			value = 0.50
+		}
+		change_relations = {
+				country = c:ATL
+				value = -15
+			}
+		set_global_variable = nouvellsatian_support
+	}
+	ai_chance = {

```

Comment:
Unexpected token: ai_chance, near line: 555" in file: "events/dvg_atlantia.txt" near line: 557

ai_chance needs to be inside option